docs/development/writing_zeppelin_interpreter.md

```diff
@@ -118,6 +118,12 @@ Here is an example of `interpreter-setting.json` on your own interpreter.
       "language": "your-syntax-highlight-language",
       "editOnDblClick": false,
       "completionKey": "TAB"
+    },
+    "config": {
+      "fontSize": 9,
+      "colWidth": 12,
+      "runOnSelectionChange": true/false,
+      "title": true/false
```

Comment:
`displayTitle` is more meaning IMO

Reply:
"fontSize", "colWidth", "runOnSelectionChange", "title"
These four parameter names need to be consistent with the variable names in paragraph-control.html.

https://github.com/apache/zeppelin/blob/master/zeppelin-web/src/app/notebook/paragraph/paragraph-control.html#L176

Reply:
Ok, after I finish this PR, I will open a PR and deal with it.